modules/util/portal-tools-db-partition-virtual-instance-migration-common/src/main/java/com/liferay/portal/tools/db/partition/virtual/instance/migration/common/InstanceData.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.tools.db.partition.virtual.instance.migration.common;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * @author Luis Ortiz
+ */
+public class InstanceData {
+
+	public InstanceData() {
+		_date = new Date(System.currentTimeMillis());
+	}
+
+	public List<Company> getCompanies() {
+		return _companies;
+	}
+
+	public Long getCompanyId() {
```

Comment:
This should be extractedCompanyId or something like that since we are returning Instance data not tie to a specific companyId